Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -147,14 +147,13 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         let lastMessageIndexPath = IndexPath(row: 0, section: 0)
         let prevMessageIndexPath = IndexPath(row: 1, section: 0)
 
-        if
-            rectForRow(at: prevMessageIndexPath).minY < contentOffset.y,
-            rowsRange.contains(prevMessageIndexPath.row) {
-            scrollToRow(at: prevMessageIndexPath, at: .top, animated: false)
+        if rectForRow(at: prevMessageIndexPath).minY < contentOffset.y,
+           rowsRange.contains(prevMessageIndexPath.row) {
+            scrollToRow(at: prevMessageIndexPath, at: .bottom, animated: animated)
         }
         
         if rowsRange.contains(lastMessageIndexPath.row) {
-            scrollToRow(at: lastMessageIndexPath, at: .top, animated: animated)
+            scrollToRow(at: lastMessageIndexPath, at: .bottom, animated: animated)
```

Comment:
the same here as https://github.com/GetStream/stream-chat-swift/pull/1314/files#r695162879